compass/landice/tests/ismip6_forcing/ismip6_forcing.cfg

```diff
@@ -0,0 +1,58 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_ais]
+
+# Base path to the input ismip6 ocean and smb forcing files. User has to supply.
+base_path_ismip6 = NotAvailable
+
+# Base path to the the MALI mesh. User has to supply.
+base_path_mali = NotAvailable
+
+# Forcing end year of the ISMIP6 data. User has to supply.
+# Available end years are 2100 and 2300.
+period_endyear = NotAvailable
+
+# Base path to which output forcing files are saved.
+output_base_path = NotAvailable
+
+# Name of climate model name used to generate ISMIP6 forcing data. User has to supply.
+# Available model names are the following: CCSM4, CESM2-WACCM, CSIRO-Mk3-6-0, HadGEM2-ES, NorESM1-M, UKESM1-0-LL
```

Comment:
I think we support `CESM2` (with end year 2100) in addition to `CESM2-WACCM` (with end year 2300).  That should be listed here and the documentation updated accordingly